pages/success-banner/index.js

```diff
@@ -1,13 +1,47 @@
 import React from 'react';
 import Head from 'next/head';
 
-class Button extends React.Component {
+const eventLog = (eventDesc) => {
+  const data = 'Banner ' + eventDesc;
+
+  return () => {
+    const url = new URL(location.href);
+
+    if (url.searchParams.get('etcher-version').indexOf('1.0.0') !== -1) {
```

Comment:
Does this mean that the current 'development' versions of Etcher that we're building and testing with (which include these new Robot-logging changes) ought to have a version-number different to `1.0.0`?
Ping @jviotti I guess this is (almost) an example of "API version-breaking" ;)